tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/DataManagement/Courses/UploadTests.cs

```diff
@@ -298,5 +298,229 @@ private MultipartFormDataContent CreateMultiPartDataContent(string contentType,
             }
             return content;
         }
+
```

Comment:
As above, each of the rows created here need to have all the required columns specified.